scalastyle-config.xml

```diff
@@ -10,12 +10,7 @@
     <check level="warning" class="org.scalastyle.scalariform.SpacesAfterPlusChecker" enabled="true"/>
     <check level="warning" class="org.scalastyle.file.WhitespaceEndOfLineChecker" enabled="true"/>
     <check level="warning" class="org.scalastyle.scalariform.SpacesBeforePlusChecker" enabled="true"/>
-    <check level="warning" class="org.scalastyle.file.FileLineLengthChecker" enabled="true">
```

Comment:
I think we should rather disable than delete this. We should have a line length check of perhaps a bit longer.